src/tools/genpolicy/rules.rego

```diff
@@ -55,13 +55,110 @@ default AllowRequestsFailingPolicy := false
 S_NAME_KEY = "io.kubernetes.cri.sandbox-name"
 S_NAMESPACE_KEY = "io.kubernetes.cri.sandbox-namespace"
 BUNDLE_ID = "[a-z0-9]{64}"
+# from https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names
+# and https://github.com/kubernetes/kubernetes/blob/8294abc599696e0d1b5aa734afa7ae1e4f5059a0/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L177
+SUBDOMAIN_NAME = "^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"
+ALLOWED_SUBDOMAIN_NAMES := ["$(host-name)", "$(node-name)", "$(pod-uid)"]
+ALWAYS_ALLOWED = ["$(resource-field)", "$(todo-annotation)"]
 
 CreateContainerRequest:= resp {
   resp = CreateContainerRequestCommon(input.base)
 
+  i_env_map := input.env_map
+
+  some p_container in policy_data.containers
```

Comment:
Look into directly matching against specific container, as opposed to checking all

Reply:
Maybe 
input.OCI.Annotations["io.kubernetes.cri.sandbox-name"] == p.OCI.Annotations["io.kubernetes.cri.sandbox-name"]

input.OCI.Annotations["io.kubernetes.cri.container-name"] == p.OCI.Annotations["io.kubernetes.cri.container-name"]

Reply:
Look into if k8s guarantees startup order from the YAML (separate PR)